koku/api/settings/test/cost_groups/test_query_handler.py

```diff
@@ -76,6 +77,16 @@ def test_get_cost_groups_filters(self):
                     for item in data:
                         self.assertEqual(item.get(filter_option), filter_value)
 
+    def test_get_cost_groups_filter_cluster(self):
+        """Basic test to exercise the API endpoint"""
+        param = {"filter[cluster]": OCP_ON_GCP_CLUSTER_ID}
+        with schema_context(self.schema_name):
+            response = self.client.get(self.url, param, headers=self.headers)
```

Comment:
The `headers` parameter wasn't added until Django 4.2. I'm getting ahead of myself.
```suggestion
            response = self.client.get(self.url, param, **self.headers)
```